tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -1663,9 +1683,12 @@ HRESULT CInstrumentationMethod::ExceptionUnwindFunctionEnter(
 
     CComPtr<IProfilerManagerLogging> spLogger;
     m_pProfilerManager->GetLoggingInstance(&spLogger);
-    spLogger->LogDumpMessage(_T("<ExceptionUnwindFunctionEnter>"));
-    spLogger->LogDumpMessage(bstrFullName);
-    spLogger->LogDumpMessage(_T("</ExceptionUnwindFunctionEnter>\r\n"));
+
+    tstring strMessage(_T("    <ExceptionUnwindFunctionEnter>"));
```

Comment:
Maybe a helper, like WriteTag(string buffer, tagName, indentLevel, optionalValue)?

Reply:
I think I'll leave it as-is for now and follow up with another change. I think there's more opportunity for XML logging unification using the CLogging::XmlDumpHelper class.